sql/src/test/java/io/crate/integrationtests/AnyIntegrationTest.java

```diff
@@ -31,13 +30,8 @@
 
 import static org.hamcrest.Matchers.is;
 
-@CrateIntegrationTest.ClusterScope(scope = CrateIntegrationTest.Scope.GLOBAL)
 public class AnyIntegrationTest extends SQLTransportIntegrationTest {
 
-    static {
-        ClassLoader.getSystemClassLoader().setDefaultAssertionStatus(true);
```

Comment:
don't we force assertions this way anymore? are they enabled in a different way now? if not maybe moving this to SQLTransportIntegrationTest...